data/templates/default.context

```diff
@@ -23,6 +23,7 @@ $endif$
   style=$linkstyle$,
   color=$linkcolor$,
   contrastcolor=$linkcontrastcolor$]
+\setupurl[style=]
```

Comment:
shouldn't this have a variable in it?

Reply:
That is, `\setupurl[style=$urlstyle$]`?

Reply:
It depends what we want. If we just want to make the smallest possible change, and have all links be styled by `$linkstyle$` then we don't need a variable there.

Alternatively, If we want to add support for styling links without link text, like automatic link URLs, then I think that would be the way to do it. I would need to add the `$urlstyle$` variable to the manual too.

Reply:
I think adding the variable would be consistent with what we do in the latex template:
```
\urlstyle{$if(urlstyle)$$urlstyle$$else$same$endif$}
```
`urlstyle` is documented in the manual under "Variables for LaTeX."  One could put something similar under the ConTeXt heading.

Reply:
OK, I have done that.